docs/quick-start.md

```diff
@@ -53,3 +53,96 @@ python manage.py update_vector_indexes
 ```
 
 To skip the prompt, use the `--noinput` flag.
+
+## Using event-stream (WagtailVectorIndexSSEConsumer)
+
+The WagtailVectorIndexSSEConsumer is a asynchronous HTTP consumer designed for handling Server-Sent Events (SSE), for streaming responses from queries using the vector index in real-time. Using the consumer requires ASGI (uvicorn, daphne etc.) along with configuring django-channels.
+
+Configure channels via this [guide](https://channels.readthedocs.io/en/stable/topics/channel_layers.html#configuration), otherwise, simply add channels to INSTALLED_APPS.
+
+```python
+INSTALLED_APPS = [
+    "channels",
+    # ...
+]
+```
+
+and set the channel layer. The [in-memory channel layer](https://channels.readthedocs.io/en/stable/topics/channel_layers.html#in-memory-channel-layer) can be used for local envrionments, however, redis is the only offically maintained channel layer supported for production use, thus [channels_redis](https://pypi.org/project/channels-redis/) will have to be install.
```

Comment:
**question:** Do we need to use channel layers? I can't see them used in the consumer

Reply:
Removed, Thanks for spotting this one. I think I added channels in when experimenting with communicating to the consumer from a sync view and assumed it has been a requirement since.